samples/mesh/main.c

```diff
@@ -301,33 +301,30 @@ static void init_shader(void)
     p = pb_begin();
 
     /* Set run address of shader */
-    pb_push(p++, NV097_SET_TRANSFORM_PROGRAM_START, 1); *(p++)=0;
+    p=pb_push1(p, NV097_SET_TRANSFORM_PROGRAM_START, 0);
 
     /* Set execution mode */
-    pb_push1(p, NV097_SET_TRANSFORM_EXECUTION_MODE,
+    p=pb_push1(p, NV097_SET_TRANSFORM_EXECUTION_MODE,
         MASK(NV097_SET_TRANSFORM_EXECUTION_MODE_MODE, NV097_SET_TRANSFORM_EXECUTION_MODE_MODE_PROGRAM)
         | MASK(NV097_SET_TRANSFORM_EXECUTION_MODE_RANGE_MODE, NV097_SET_TRANSFORM_EXECUTION_MODE_RANGE_MODE_PRIV));
-    p += 2;
 
-    pb_push1(p, NV097_SET_TRANSFORM_PROGRAM_CXT_WRITE_EN, 0);
-    p += 2;
+    p=pb_push1(p, NV097_SET_TRANSFORM_PROGRAM_CXT_WRITE_EN, 0);
 
     /* Set cursor and begin copying program */
-    pb_push1(p, NV097_SET_TRANSFORM_PROGRAM_LOAD, 0);
-    p += 2;
+    p=pb_push1(p, NV097_SET_TRANSFORM_PROGRAM_LOAD, 0);
 
     for (i=0; i<sizeof(vs_program)/8; i++) {
         pb_push(p++, NV097_SET_TRANSFORM_PROGRAM, 4);
         memcpy(p, &vs_program[i*4], 4*4);
         p+=4;
     }
 
-    pb_end(p);
+    p=pb_end(p);
 
     /* Setup fragment shader */
     p = pb_begin();
     #include "ps.inl"
-    pb_end(p);
+    p=pb_end(p);
```

Comment:
Ok but not really necessary. We never use this pointer after here. Same comment for all instances.

Reply:
The weird thing here is also the different formatting of the `pb_begin` and `pb_end` line.

See my other comment for rationale of `pb_end` return value.

Reply:
`p = pb_*` formatting fixed throughout the file.